libraries/entities/src/ShapeEntityItem.cpp

```diff
@@ -299,11 +305,16 @@ bool ShapeEntityItem::findDetailedRayIntersection(const glm::vec3& origin, const
 }
 
 bool ShapeEntityItem::findDetailedParabolaIntersection(const glm::vec3& origin, const glm::vec3& velocity, const glm::vec3& acceleration,
-                                                       OctreeElementPointer& element, float& parabolicDistance,
+                                                       const glm::vec3& viewFrustumPos, OctreeElementPointer& element, float& parabolicDistance,
                                                        BoxFace& face, glm::vec3& surfaceNormal,
                                                        QVariantMap& extraInfo, bool precisionPicking) const {
+    glm::vec3 dimensions = getScaledDimensions();
+    glm::quat rotation = getWorldOrientation();
+    glm::vec3 position = getWorldPosition() + rotation * (dimensions * (ENTITY_ITEM_DEFAULT_REGISTRATION_POINT - getRegistrationPoint()));
```

Comment:
Does billboarding rotate about the centre of the entity of its registration point?
Which is the desired behaviour?

Reply:
I *think* the old code billboarded around the center of the rendering, so if you offset it with registrationPoint, it would face you from that point.  I think this is expected, so I’ve tried to carry that logic through.  I’m definitely open to thoughts on this.

You can test this with the test script by pressing ‘k’ to toggle through 3 different registration points.

Reply:
Tested 2020.3.3 and PR build: text entity with 0, 0, 0 registration point pivots about the centroid of the entity (not about the entity position / registration point. 

Tested Shape entity in PR build: shape entity with 0, 0, 0 registration point. It pivots about its centroid. So consistent behaviour.